### PR TITLE
Cluster now default, simplified code

### PR DESCRIPTION
**Pull request by geronimp**

### CODE REVIEW

**Comment by wwood** [comment]
I'm done for the moment. Lemme know when you are happy and I'll test.

**Comment by geronimp** [comment]
how 'bout now?

File: graftm/run.py [hunk 1]
```diff
@@ -367,26 +363,24 @@ def graft(self):
                                self.args.output_directory,
                                False)
 
+
+        # Classification steps
```
Comment (wwood) [inline graftm/run.py]: shouldn't this be inside the next `if` statement? Or so you want to cluster before diamond as well?

Comment (geronimp) [inline graftm/run.py]: No, diamond is a little trickier and I don't really want to go into that part of the code just yet tbh. I just needed the clustering in the HMM side of things for timing.


I've moved the code block into the if statement.


________________________________

**Comment by wwood** [comment]
Fail on the testing I'm afraid. I also notice that this `hit_read_counts` is only referenced in a place that returns separately, above, on line 821. Be good to report the number of seqs though, can then close an issue.
```
ben@u:~/git/graftM$ nosetests -x
E
======================================================================
ERROR: test_hello_world (test_bootstrapper.Tests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ben/git/graftM/test/test_bootstrapper.py", line 46, in test_hello_world
    tf.name))
  File "/home/ben/git/graftM/graftm/bootstrapper.py", line 82, in generate_hmm_from_contigs
    hit_reads_orfs_fasta.name)
  File "/home/ben/git/graftM/graftm/hmmer.py", line 844, in search_and_extract_orfs_matching_protein_database
    logging.info("%s read(s) detected" % hit_read_counts[1])
UnboundLocalError: local variable 'hit_read_counts' referenced before assignment
-------------------- >> begin captured logging << --------------------
root: INFO: Finding bootstrap hits in provided contigs..
root: DEBUG: Finding bootstrap hits in /home/ben/git/graftM/test/data/bootstrapper/contigs.fna..
root: DEBUG: Detected file format FORMAT_FASTA
root: DEBUG: raw read unpacking command chunk: cat /home/ben/git/graftM/test/data/bootstrapper/contigs.fna
root: DEBUG: Detected sequence type as nucleotide
root: DEBUG: Using 1 HMMs to search
root: DEBUG: OrfM command chunk: orfm  /home/ben/git/graftM/test/data/bootstrapper/contigs.fna
root: DEBUG: Running command: orfm  /home/ben/git/graftM/test/data/bootstrapper/contigs.fna | hmmsearch --domE 1e-5 --cpu 1 -o /dev/null --noali --domtblout /tmp/graftm_bootstrap2esjcY0 /home/ben/git/graftM/test/data/bootstrapper/DNGNGWU00001.hmm -
root: DEBUG: Running extern cmd: orfm  /home/ben/git/graftM/test/data/bootstrapper/contigs.fna | hmmsearch --domE 1e-5 --cpu 1 -o /dev/null --noali --domtblout /tmp/graftm_bootstrap2esjcY0 /home/ben/git/graftM/test/data/bootstrapper/DNGNGWU00001.hmm -
root: DEBUG: Running extern cmd: fxtract -H -X -f /tmp/graftm_readnamesurq_Tm  /home/ben/git/graftM/test/data/bootstrapper/contigs.fna > /tmp/_raw_extracted_reads.faanqsrr
root: DEBUG: OrfM command chunk: orfm  
--------------------- >> end captured logging << ---------------------

----------------------------------------------------------------------
Ran 1 test in 0.650s

FAILED (errors=1)
```

**Comment by geronimp** [comment]
Should pass tests now.

**Pull request merged**